unified_planning/io/pddl_writer.py

```diff
@@ -1061,22 +1075,24 @@ def _write_effect(
     if not simplified_cond.is_true():
         out.write(f" (when {converter.convert(effect.condition)}")
     simplified_value = effect.value.simplify()
+    fluent = converter.convert(effect.fluent)
     if simplified_value.is_true():
-        out.write(f" {converter.convert(effect.fluent)}")
+        out.write(f" {fluent}")
     elif simplified_value.is_false():
-        out.write(f" (not {converter.convert(effect.fluent)})")
+        out.write(f" (not {fluent})")
+    elif initial:
+        value = converter.convert(simplified_value)
+        if effect.is_increase():
+            value = f"(+ {fluent} {value})"
```

Comment:
This does not seem to be mentionned in the PDDL 2.2 references (that I believe assume literals in TIL). Do we have any example of this syntax usage ? (I am surprised because I would have assumed `increase`/`decrease` to be used there).

Reply:
This is not in the standard, both in this way or with the `increase`/`decrease`.
Maybe it is better to raise an exception. What do you think?

Reply:
I have looked around a bit and only found one in a [short paper](https://kclpure.kcl.ac.uk/portal/en/publications/planning-with-numeric-timed-initial-fluents). They only consider assignment to numeric fluents, no increase/decrease.

I think their proposal to use `=` for assignments in timed effects is misleaded and that timed effects should use exactly the same syntax as in regular action effects. That is one of:

    (at <time-spec> <l>)
    (at <time-spec> (not <l>))
    (at <time-spec> (assign <sv> <v>))
    (at <time-spec> (increase <sv> <v>))
    (at <time-spec> (decrease <sv> <v>))


That way there is no new syntax introduced for timed effects. They only differ from action effects in what is allowed for <time-spec>: strictly positive number for TILs and `start`/`end` for actions.

I believe it is fine to introduce such very minor syntax extensions as long as (1) it solves a common problem, (2) it is perfectly unambiguous and (3) we are convinced that the syntax is future-proof. I think the above proposal matches all three.


(Note that we are still missing the `scale-up`/`scale-down` modifiers from PDDL 2.1)

Reply:
I think your proposal is reasonable. I updated the code accordingly.

Where do you think we should document this extension?